pallets/mixer/src/tests.rs

```diff
@@ -99,38 +90,29 @@ fn should_deposit_into_each_mixer_successfully() {
 fn should_withdraw_from_each_mixer_successfully() {
 	new_test_ext().execute_with(|| {
 		assert_ok!(Mixer::initialize(Origin::signed(1)));
-		let mut test_rng = rand::thread_rng();
-		let h = default_hasher();
 		let pc_gens = PedersenGens::default();
-		let bp_gens = BulletproofGens::new(4096, 1);
+		let poseidon = default_hasher(40960);
```

Comment:
Did you mean to make this 4096?

Reply:
Since we now have `57` partial round in Poseidon hash function, the number of generators goes up really high, so we need around 40k for 32h tree